README.md

```diff
@@ -95,6 +95,94 @@ status:
   state: DEPLOYED
 ```
 
+## Minimum resource requirement
+
+The minimum memory and cpu requirements for each ixia-c components are captured in the following table. Kubernetes metrics server has been used to collect resource usage data. The memory represents the minimum working set memory required, and for protocol and traffic engines, varies depending on the number of colocated ports. The figures are in Mi or MB.
+
+                | --------------- | ------- | ------- | ------- | ------- | ------- |
+                | Component       | 1 Port  | 2 Port  | 4 Port  | 6 Port  | 8 Port  |
+                |                 | Default |         |         |         |         |
+                | --------------- | ------- | ------- | ------- | ------- | ------- |
+                | Protocol Engine |   350   |   420   |   440   |   460   |   480   |
+                | --------------- | ------- | ------- | ------- | ------- | ------- |
+                | Traffic Engine  |    60   |    70   |    90   |   110   |   130   |
+                | --------------- | ------- | ------- | ------- | ------- | ------- |
+                | Controller      |    25   |         |         |         |         |
+                | --------------- | ------- | ------- | ------- | ------- | ------- |
+                | gNMI            |    15   |         |         |         |         |
```

Comment:
Maybe a * can be added with a clarification that gNMI and Controller have a fixed minimum memory requirement and is currently not dependent on number of test ports for the topology.

Reply:
Added footnote